tests/db_tests/test_db.py

```diff
@@ -0,0 +1,60 @@
+from conftests import task_data
+
+from qhana_plugin_runner.db.models.tasks import ProcessingTask
+from qhana_plugin_runner.db.models.mutable_json import (
+    TrackedDict,
+    TrackedList,
+)
+
+
+def test_json_strorage(task_data: ProcessingTask):
+    # Test mutable json data attribute
+    assert task_data.data == dict()
```

Comment:
please add descriptions for when the assertion fails:
```suggestion
    assert task_data.data == dict(), "Task data.data is expected to be an empty dict at the start of the unit test."
```